modules/ROOT/pages/custom-dictionaries-for-tiny-spellchecker.adoc

```diff
@@ -1,11 +1,11 @@
 = Adding custom dictionaries
-:description: Custom dictionaries can be added to Spell Checker Pro
-:keywords: enterprise, tinymcespellchecker, spell check, spell checker pro
+:description: Custom dictionaries can be added to Spell Checker
+:keywords: enterprise, tinymcespellchecker, spell check, Spell Checker
```

Comment:
I am assuming this doesn't necessarily have to be lower case?

Reply:
```suggestion
:keywords: enterprise, tinymcespellchecker, spell check, spell checker
```